application/Gemfile

```diff
@@ -48,11 +40,10 @@ group :development, :test do
 end
 
 group :development do
+  # Use the Puma web server [https://github.com/puma/puma]
+  gem "puma", ">= 5.0"
   # Use console on exceptions pages [https://github.com/rails/web-console]
   gem "web-console"
-
-  # Highlight the fine-grained location where an error occurred [https://github.com/ruby/error_highlight]
-  gem "error_highlight", ">= 0.4.0", platforms: [ :ruby ]
 end
```

Comment:
I guess this gem was bugging you and you removed because of that. During my analysis I had also to remove this gem